context/brand-context/default/scss/10-settings/buttons.scss

```diff
@@ -19,6 +19,8 @@ $button--padding: spacing(8);
 $button--padding-large: spacing(16);
 $button--justify-content: center;
 $button--transition: 0.25s ease, color 0.25s ease, border-color 0.25s ease;
+$button--border-radius: null;
```

Comment:
these are specific to `springernature`, setting them to null be default means they won't be used for other brands